doc/examples/gerchberg_saxton.py

```diff
@@ -7,11 +7,11 @@
 
 import numpy as np
 
-from light_prop.algorithms import GerchbergSaxton
-from light_prop.calculations import gaussian, get_gaussian_distribution
-from light_prop.lightfield import LightField
-from light_prop.propagation.params import PropagationParams
-from light_prop.visualisation import GeneratePropagationPlot, PlotTypes
+from lightprop.algorithms import GerchbergSaxton
+from lightprop.calculations import gaussian, get_gaussian_distribution
+from lightprop.lightfield import LightField
```

Comment:
```suggestion
from lightprop.lightfield.lightfield import LightField
```
sounds weird btw `lightfield.lightfield`